### PR TITLE
Tao 5547/specify language for tts in items

### PR DESCRIPTION
**Pull request by IlyaYarkavetsOnePoint**
This additions are within TAO-5547 task. New handlebars helper function is added. It allows to check equation of two variables.
Syntax:
```
{{#ifequals var1 var2}}
    do_something
{{/if}}
```

### CODE REVIEW

**Comment by siwane** [comment]
Could u assign the PR to a frontend guy please

File: views/js/lib/handlebars.js [hunk 1]
```diff
@@ -312,6 +312,16 @@ var __module2__ = (function(__dependency1__, __dependency2__) {
       var level = options.data && options.data.level != null ? parseInt(options.data.level, 10) : 1;
       instance.log(level, context);
     });
+
+    instance.registerHelper('ifequals', function(lvalue, rvalue, options) {
+      if (arguments.length < 3)
+          throw new Error("Handlebars Helper equal needs 2 parameters");
+      if( lvalue!=rvalue ) {
```
Comment (jsconan) [inline views/js/lib/handlebars.js]: Please use strict operators: `===` or `!==`. Simple comparison operators should not be used, unless you rely on their sick behavior for performance purpose.

File: views/js/lib/handlebars.js [hunk 2]
```diff
@@ -312,6 +312,16 @@ var __module2__ = (function(__dependency1__, __dependency2__) {
       var level = options.data && options.data.level != null ? parseInt(options.data.level, 10) : 1;
       instance.log(level, context);
     });
+
+    instance.registerHelper('ifequals', function(lvalue, rvalue, options) {
```
Comment (jsconan) [inline views/js/lib/handlebars.js]: Maybe should you use a simpler name, like `equals`.

**Comment by IlyaYarkavetsOnePoint** [comment]
> For Handlebars, there is already a module to add new helpers, please use it
Thanks for the info! Didn't know that the one already exists.

Closing this pr as another helper with same function already exists.

**Pull request closed without merging**